assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -1,26 +1,27 @@
-FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu117-py38-torch201:{{latest-image-tag:biweekly.\d{6}.\d{1}}}
+FROM mcr.microsoft.com/aifx/acpt/stable-ubuntu2004-cu118-py310-torch201:latest
+
+# Install basic dependencies and pip
+RUN apt-get update
 
 WORKDIR /
 
 # support Deepspeed launcher requirement of passwordless ssh login
 RUN apt-get update && apt-get -y upgrade
-RUN apt-get install -y openssh-server openssh-client
-
-COPY requirements.txt .
-RUN pip install -r requirements.txt --no-cache-dir
-
-# List installed packages
-RUN pip list
 
-## Delete
-RUN rm requirements.txt
+COPY . .
+RUN pip install --no-cache-dir -e .
+RUN pip list    # show installed packages
 
 # Copy scoring file
-COPY score.py /var/mlflow_resources/mlflow_score_script.py
+COPY src/score.py /var/mlflow_resources/mlflow_score_script.py
```

Comment:
Dont need to copy all src files to mlflow_resources?

Reply:
No. 
So right now, that since score.py file we were used to is a full python project. L:11 is copying over the whole source, and doing the editable installation of our code. The score.py is just going to be a thin facade over that code (progressively)